sdk/src/main/java/io/savantlabs/stylus/sdk/jupyter/JupyterClientImpl.java

```diff
@@ -27,4 +32,60 @@ public class JupyterClientImpl implements JupyterClient {
   public void stop() {
     process.stop();
   }
+
+  @Override
+  public void installPackage(String installPkgsList) throws InterruptedException {
+    Process process =
+        ShellRunner.runScript(
+            (p) ->
+                (line) -> {
+                  log.info("[Install] "+line);
+                },
+            (p) ->
+                (line) -> {
+                },
+            "install_pkg.sh",installPkgsList);
+    process.waitFor();
```

Comment:
I think the default consumer (simple logs) works fine, no need to add customized consumers.
But better add a validation after `process.waitFor`, in case the process exits, but the package was not installed due to some error.

At least verify the process exit code is 0.